content/en/docs/collector/building/authenticator-extension.md

```diff
@@ -1,6 +1,7 @@
 ---
-title: Building a custom authenticator
-weight: 30
+title: Building an authenticator extension
```

Comment:
An authenticator is an extension. In fact, I think the previous title still resonates better with me.

Reply:
```suggestion
title: Building an authenticator
```

Reply:
works for me as well, do you want the "custom" still be there as well?

Reply:
I have a slight preference for the version with "custom", but they are both fine.

Reply:
I will leave "custom" out more because it makes the navbar links a little shorter